src/help/components/results/styles/highlight.scss

```diff
@@ -1,6 +1,6 @@
 @import 'franklin-sites/src/styles/colours';
 
-.help-card :global(.match-highlight) {
+.match-highlight {
```

Comment:
Does this not need `:global` ?

Reply:
This is because it's not a CSS module file anymore, so the class name wouldn't be changed.

The reason I put a CSS module class name in the original rule was to make sure to scope this rule somehow, in case something else somewhere else in the logic would end up having the same class name injected.

Reply:
As `highlight.scss` is now imported only within `CleanHighlightMarkDown.tsx` this should be OK right or am I polluting the global namespace?

Reply:
No that's ok.